pom.xml

```diff
@@ -1,30 +1,116 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!--
-  Licensed under the Apache License, Version 2.0 (the "License");
-  you may not use this file except in compliance with the License.
-  You may obtain a copy of the License at
-
-    http://www.apache.org/licenses/LICENSE-2.0
-
-  Unless required by applicable law or agreed to in writing, software
-  distributed under the License is distributed on an "AS IS" BASIS,
-  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-  See the License for the specific language governing permissions and
-  limitations under the License. See accompanying LICENSE file.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License. See accompanying LICENSE file.
 -->
 <project xmlns="http://maven.apache.org/POM/4.0.0"
-  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0
-                      http://maven.apache.org/maven-v4_0_0.xsd">
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0
+http://maven.apache.org/maven-v4_0_0.xsd">
+  <parent>
+    <groupId>org.apache.commons</groupId>
+    <artifactId>commons-parent</artifactId>
+    <version>39</version>
+  </parent>
   <modelVersion>4.0.0</modelVersion>
-  <groupId>com.intel.chimera</groupId>
-  <artifactId>chimera</artifactId>
+  <groupId>commons-crypto</groupId>
+  <artifactId>commons-crypto</artifactId>
   <packaging>jar</packaging>
-  <description>chimera: A fast encryption/decryption library</description>
+  <description>
+    Apache Commons Crypto is a cryptographic library optimized with AES-NI (Advanced Encryption
+    Standard New Instructions). It provides Java API for both cipher level and Java stream level.
+    Developers can use it to implement high performance AES encryption/decryption with the minimum
+    code and effort. Please note that Crypto doesn't implement the cryptographic algorithm such as
+    AES directly. It wraps to Openssl or JCE which implement the algorithms.
+  </description>
   <version>1.0.0-SNAPSHOT</version>
-  <name>chimera</name>
-  <url>https://github.com/intel-hadoop/chimera</url>
-
+  <name>Apache Commons Crypto</name>
+  <inceptionYear>2016</inceptionYear>
+  <url>http://commons.apache.org/proper/commons-crypto</url>
+  <issueManagement>
+    <system>jira</system>
+    <url>http://issues.apache.org/jira/browse/CRYPTO</url>
+  </issueManagement>
+  <scm>
+    <connection>
```

Comment:
SCM information is not correct. We use git instead svn. So it should be:
<scm>
  <connection>scm:git:http://git-wip-us.apache.org/repos/asf/commons-crypto.git</connection>
  <developerConnection>scm:git:https://git-wip-us.apache.org/repos/asf/commons-crypto.git</developerConnection>
  <url>https://git-wip-us.apache.org/repos/asf?p=commons-crypto.git</url>
</scm>